src/databricks/labs/ucx/install.py

```diff
@@ -906,6 +906,21 @@ def _repair_workflow(self, workflow):
         run_id = latest_job_run.run_id
         return job_id, run_id
 
+    def map_cluster_to_uc(self, cluster_id):
```

Comment:
it doesn't belong to `WorkspaceInstallation` class. for now, create the `workspace_access/clusters.py` and add this remapping logic there. we'll be expanding that class a lot